docs/faq.md

```diff
@@ -1,21 +1,321 @@
+
+# Technical
+
+## What languages can be used to write functions on Fn?
+
+Out of the box support includes: Java, Go, Ruby, Python, PHP, Rust, .NET Core, and Node.js (including AWS Lambda compatibility).
+
+Since we use containers as the base building block, all languages can be used. There may not be higher level helper libraries like our Lambda wrapper for every language, but you can use any language if you follow the base function format.
+
+## What platforms does Fn run on?
+
+Any Linux/Unix platform with Docker support which includes Linux, MacOS, and FreeBSD.
+
+## What software do I need to have installed as a prerequisite for running Fn?
+
+You’ll need a recent release of Docker installed.
+
+## Why do I need Docker installed to run Fn?
+
+Fn packages functions as Docker containers which are published to a Docker registry for deployment.  The Fn server pulls images from a Docker registry when functions are invoked.
 
 ## Where can I run Fn?
 
-Anywhere. Any cloud, on-premise, on your laptop. As long as you can run a Docker container, you can run Fn.
+Anywhere. Any cloud, on-premise, on your laptop, even on AWS or Azure. As long as you can run a Docker container, you can run Fn. 
+
+## Do I need any accounts to run Fn?
+
+You don’t need any special accounts to run Fn locally.  But to deploy functions to a remote Fn server you’ll need access to a Docker registry and an account.  Docker Hub is the default registry.
 
 ## Which orchestration tools does functions support?
 
 Functions can be deployed using any orchestration tool.
 
-## Does Fn require Docker?
+## Can I deploy functions with a custom runtime?
+
+Yes, as Fn packages and deploys all functions as Docker containers it’s possible to provide a custom Docker image that includes your function.  As long as your container implements the Fn function contract it can contain anything.
+
+## What is the lifecycle of a function?
+
+Functions are packaged as Docker images and by default individual containers are created to handle a function request and are then destroyed.  However, `Hot Functions` are not disposed of after handling a single request.
+
+## What’s a `Hot Function`?
+
+`Hot Functions` are functions that are not destroyed after a single use but are retained and used to handle subsequent requests.  Hot Functions must accept HTTP input.
+
+# General
+
+## What is the Fn Project’s license?
+
+Apache 2.0
+
+## What is the URL of the project website?
+
+<http://fnproject.io>
+
+## Where is the source code hosted?
+
+All of the project code is on Github at <http://github.com/fnproject>.
+
+## Is Fn an entirely new platform?
+
+No, Fn is derived from the well-received IronFunctions project.  The core IronFunctions team are now at Oracle so Fn is simply the next evolutionary step of IronFunctions built by the original developers.
+
+## How do you pronounce “Fn”?
```

Comment:
IMO, I don't we should promote the alternative way it's pronounced, let people figure it out on their own.

Reply:
Removed funny FN comment.